SmartWeights/SmartWeights/SmartWeightsApp.swift

```diff
@@ -12,7 +12,8 @@ struct SmartWeightsApp: App {
     let persistenceController = PersistenceController.shared
     var body: some Scene {
         WindowGroup {
-            LoginView()
+            //bleView(ble: BLEcentral())
+            WorkoutMainPage()
```

Comment:
remember to change this when we merge to main or you can change it now

LoginView() is what we want